src/style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.active__star {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
No, use pseudoclasses for that purpose, no need to set class to each star that should be highlighted